gui-libs/neatvnc/neatvnc-0.2.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="liberally licensed VNC server library with a clean interface"
+
+HOMEPAGE="https://github.com/any1/neatvnc/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/any1/neatvnc.git"
+else
+	SRC_URI="https://github.com/any1/neatvnc/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="ISC"
+SLOT="0"
+IUSE="examples ssl jpeg tracing"
+
+DEPEND="
+	x11-libs/pixman
+	x11-libs/libdrm
+	dev-libs/aml
+	sys-libs/zlib
+	ssl? ( net-libs/gnutls )
```

Comment:
Does the package only support gnutls?

Reply:
unfortunately yes.
seems to be a hard requirement.

Reply:
as seen here:  https://github.com/any1/neatvnc/blob/master/meson.build#L45

I'm am a bit sad as well given recent security problems with gnutls

Reply:
also, add `:=` to gnutls and `libjpeg-turbo`

Reply:
done